pkg/config/config.go

```diff
@@ -890,6 +895,10 @@ func (c *NetworkConfig) Validate() error {
 		}
 	}
 
+	if c.DnsBindPort < 0 || c.DnsBindPort > 65535 {
+		return errors.New("DnsBindPort must be a valid port (16 bit integer)")
+	}
```

Comment:
If we change the datatype to unit16 I don't think we have to check for this since the toml parser does it.

Reply:
I've confirmed this manually, I get
```
Expected
      <*fmt.wrapError | 0xc0003a2100>: {
          msg: "reading user config \"testdata/containers_default.conf\": decode configuration testdata/containers_default.conf: toml: value 1111153 is out of range for uint16",
          err: <*fmt.wrapError | 0xc0003a20e0>{
              msg: "decode configuration testdata/containers_default.conf: toml: value 1111153 is out of range for uint16",
              err: <*errors.errorString | 0xc000150870>{
                  s: "toml: value 1111153 is out of range for uint16",
              },
          },
      }
  to be nil
  In [It] at: /home/martinet/code/containers/common/pkg/config/config_local_test.go:481
```
so unfortunately it doesn't say it's for `dns_bind_port` or line number in toml file (which my version didn't have either, anyway), but with the explicit value it should be easy enough to find. Thanks!